futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -921,3 +922,22 @@ class AggregatedCountsSerializer(ReadOnlySerializer):
     all_tenants = AggregatedCountsAllTenantsSerializer()
     by_tenant = AggregatedCountsOneTenantSerializer(many=True)
     limited_access = serializers.BooleanField()
+
+
+class FileUploadSerializer(ReadOnlySerializer):
+    """
+    Serializer for handling the file upload request. It validates and serializes the input data.
+    """
+    file = serializers.FileField()
+    slug = serializers.CharField(max_length=255)
```

Comment:
to prevent invalid characters
```suggestion
    slug = serializers.SlugField()
```